app/views/cars/new.html.erb

```diff
@@ -1,15 +1,35 @@
 <div class="container">
-  <div class="form_container shadow">
+  <div class="add-car-form shadow rounded">
+    <h1 class="fugaz-font m-3" style="font-size: 24px; text-align: center;">Add your car</h1>
     <%= simple_form_for(@car) do |f| %>
-      <%= f.input :model %>
-      <%= f.input :year %>
-      <%= f.input :seats %>
-      <%= f.input :colour %>
-      <%= f.input :address %>
-      <%= f.input :price %>
-      <%= f.input :description %>
+    <div class="row">
+      <div class="col">
+      <%= f.input :model, required: false %>
```

Comment:
What does adding required: false do here?
Users should be able to make a car without adding these.

Reply:
it removes the .* after the name field label

I've done it for most forms

Reply:
Ok. Did you test making a car without inputting one of these?
I guess validations will still stop it?